build_tools/i18n/crowdin.py

```diff
@@ -0,0 +1,396 @@
+# -*- coding: utf-8 -*-
+"""
+For usage instructions, see:
+    https://kolibri-dev.readthedocs.io/en/develop/references/i18n.html
```

Comment:
If you use docopt, it's slightly easier to get documentation, CLI instructions and parsing in one go.

Reply:
follow-up issue: https://github.com/learningequality/kolibri/issues/4364